### PR TITLE
feat: add toggle component

### PR DESCRIPTION
**Pull request by ajlu**
## Description

Add Toggle component (used to toggle **Use points** in Qantas Holidays)

💁‍

## Related issues

💪

## Looks like this

Screen Shot 2021-07-26 at 11 26 52 am

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Angus](https://github.com/angusfretwell), [Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), or [Long](https://github.com/KieraDOG)).
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [x] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by ehyland** [comment]
👷 Deploy Preview for *roo-ui* processing.


🔨 Explore the source changes: f55f703e30e4fa4ab16316b29fd7aa13edfa1f73

🔍 Inspect the deploy log: [https://app.netlify.com/sites/roo-ui/deploys/60fa55000e3da10008f80e5e](https://app.netlify.com/sites/roo-ui/deploys/60fa55000e3da10008f80e5e?utm_source=github&utm_campaign=bot_dl)

**Review by KieraDOG — APPROVED** [review]
LGTM 👍

**Review by philipwindeyer — CHANGES_REQUESTED** [review]
Niiiice 🤙  LGTM

Just missing one thing; the typedef for TypeScript projects within the org.
Component definitions live here: https://github.com/hooroo/roo-ui/blob/master/src/components/index.d.ts

If I had to guess, the `Radio` and ` Checkbox` definitions are probably the closest to what the def might look like for this

**Review by philipwindeyer — APPROVED** [review]
LGTM 🚀

File: src/components/Toggle/Toggle.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  display: inline-block;
+  position: relative;
+  width: ${rem('38px')};
+  height: ${rem('24px')};
+`;
+
+const ToggleInput = styled.input`
+  opacity: 0;
+  width: 0;
+  height: 0;
```
Comment (mrm007) [inline src/components/Toggle/Toggle.js:25]: I would recommend [`hideVisually`](https://polished.js.org/docs/#hidevisually) instead ([why]).

[why]: https://snook.ca/archives/html_and_css/hiding-content-for-accessibility

**Review by eneo5541 — APPROVED** [review]
LGTM!

File: src/components/Toggle/Toggle.js (hunk 1 above)
Comment (ajlu) [reply src/components/Toggle/Toggle.js]: Thanks! updated 😄

**Review by Anthropic — APPROVED** [review]

**Pull request merged**